### ISSUE TITLE
[Firefox] QField with clearable keeps focus state after clear + click outside

### ISSUE THREAD

**Issue by sullyD64** [quasarframework/quasar#13354]
### What happened?

Hi, i'm on Firefox 99 and it seems that the bug described in [#6906](https://github.com/quasarframework/quasar/issues/6906) is still present. I'm opening another issue as @pdanpdan asked on Discord.

For convenience, i'm pasting here the content of that issue.

---

QField with clearable prop seems to keep focus state after clicking the clear button and clicking outside the field. Only after then clicking into the field and back out will the focus appearance disappear.

**Seems to be specific to Firefox.**

This also seems to affect external validation triggering (e.g. a required rule not triggering until the field is refocused and reblurred so it does not appear to be only visual.

### What did you expect to happen?

Focus appearance should be lost when clicking outside 

### Reproduction URL

https://codepen.io/gordonblahut/pen/VwvzPKJ

### How to reproduce?

-     Go to https://codepen.io/gordonblahut/pen/VwvzPKJ **using Firefox**
-     Click on the clear icon in the first input
-     Click outside the input
-     See the input still has a blue underline to indicate focus/active

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox

### Quasar info output

```Shell
Operating System - Linux(5.13.0-39-generic) - linux/x64
NodeJs - 14.15.0

Global packages
  NPM - 8.1.0
  yarn - 1.22.17
  @quasar/cli - undefined
  @quasar/icongenie - 2.5.1
  cordova - Not installed

Important local packages
  quasar - 2.6.6 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-webpack - 3.5.1 -- Quasar Framework App CLI with Webpack
  @quasar/extras - 1.13.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.12
  pinia - Not installed
  vuex - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.17.0 -- Babel compiler core.
  webpack - 5.72.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.8.1 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - 6.5.3 -- A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - P15
  wlo1 - 192.168.1.244
  enx98fc84e413f3 - 192.168.1.13
  tun0 - 10.10.11.9
```

### Additional context

This is what happens in the codepen after clicking the "clear" button:
![immagine](https://user-images.githubusercontent.com/26333714/166884683-e3e8ccf7-d442-45ad-b49f-0994cd3f10ff.png)

**Comment by pdanpdan** [comment]
Fix will be available in v2.6.7/V1.18.11
